frontend/packages/dev-console/integration-tests/tests/git-import-flow.scenario.ts

```diff
@@ -12,14 +12,44 @@ import {
   applicationName,
   appName,
   createButton,
+  setRouting,
+  setBuildConfig,
+  setDeployment,
+  setResources,
+  setLabel,
+  selectAdvancedOptions,
+  AdvancedOptions,
+  TLSTerminationValues,
+  buildConfigObj,
+  addNavigate,
+  routingObj,
+  deploymentObj,
+  resourceLimitsObj,
+  setScaling,
+  scalingObj,
+  setSecureRoute,
+  pipelineObj,
+  setPipelineForGitFlow,
+  importFromGitHeader,
+  addApplicationInGeneral,
+  gitUrlHelper,
+  builderImageSelected,
 } from '../views/git-import-flow.view';
+import { 
+  // verifyCheckBox, 
```

Comment:
nit: we could remove this if it is not used

Reply:
Maybe in future I can re-use it,  so for that reason I commented this line, 
If it is a part of best coding practice, I will remove this line

Reply:
It is, please remove it. If you need it later it's not hard to add. If it was there before it's not hard to look up the history of the file.

We have a large codebase and we don't need "maybe some day" code.